pkg/console/service/service.go

```diff
@@ -33,18 +34,26 @@ const (
 	AppKubernetesComponentLabel = "app.kubernetes.io/component"
 )
 
+const (
```

Comment:
Better put these headers and the other helper funcs into the auth config pkg instead? They don't belong to the service logic itself.

Reply:
Done